src/org/thoughtcrime/securesms/components/AudioView.java

```diff
@@ -79,6 +79,7 @@ public void setAudio(final @NonNull AudioSlide audio, int duration)
   {
     controlToggle.displayQuick(playButton);
     seekBar.setEnabled(true);
+    seekBar.setProgress(0);
```

Comment:
this fixes the wrong scrolled/advanced seekbar on recycling the message view, but sadly due to the way AudioSlidePlayer work, there is no per-message progress that can be tracked statically, so a paused message will have its seekbar reset to initial position if the item is recycled, this would be solved when we add a audio mini-player,

overall it is better not to see old items getting a wrong seekbar than a paused audio losing its state because we scrolled to the old chat history